api.yaml

```diff
@@ -241,6 +243,10 @@ definitions:
     $ref: "https://raw.githubusercontent.com/teamdigitale/digital-citizenship-functions/v0.38.0/api/definitions.yaml#IsWebhookEnabled"
   PaginationResponse:
     $ref: "https://raw.githubusercontent.com/teamdigitale/digital-citizenship-functions/v0.38.0/api/definitions.yaml#PaginationResponse"
+  CreatedMessageWithContent:
```

Comment:
refs should have `#/` when referencing to remote definition, see [here](https://github.com/teamdigitale/italia-backend/pull/108/files#diff-9d362e7bbef98fc5f4460fccff661cb6R228)